internal/impl/kafka/integration_test.go

```diff
@@ -356,3 +364,76 @@ input:
 		integration.StreamTestOptVarSet("VAR1", ""),
 	)
 }
+
+func TestIntegrationKafkaOutputFixedTimestamp(t *testing.T) {
```

Comment:
It would be nice to maybe avoid adding an extra test. I was thinking to maybe introduce a new `StreamTestReadsMetadata()` which writes something to the output and then launches the input similar to `StreamTestOpenCloseIsolated()` and then checks if the metadata of the message read from the input contains the metadata keys and values which were passed via a (variadic) parameter.